backends/platform/3ds/osystem-graphics.cpp

```diff
@@ -113,12 +113,7 @@ void OSystem_3DS::init3DSGraphics() {
 
 void OSystem_3DS::destroy3DSGraphics() {
 	_gameScreen.free();
-	_gameTopTexture.free();
```

Comment:
Why are you removing these calls but not every ones?

Reply:
Because `_gameScreen` and `_cursor` are plain `Graphics::Surface` objects, while the others are `Sprite` objects. The client code is responsible for calling `Graphics::Surface.free()`. Since `Sprite` now calls `Sprite.free()` in its destructor, it doesn't really make sense to have it called here as well.

Reply:
Indeed, I did not notice that.